pom.xml

```diff
@@ -733,6 +734,12 @@
                 <version>5.1.48</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.mariadb.jdbc</groupId>
+                <artifactId>mariadb-java-client</artifactId>
+                <version>2.4.0</version>
```

Comment:
Let's move this dependency to presto-memsql/pom.xml because it's used only from memsql module for now.